examples/animation/main.js

```diff
@@ -172,6 +172,7 @@
                 config.currentAnim = config.anim || firstAnimation;
                 if ( config.currentAnim ) {
                     controller.play = function () {
+                        animationManager.stopAllAnimation();
```

Comment:
no "already playing" check to avoid restarting from 0 and/ord compute everything ?
(Or do it only if dirty or smth ?)